modules/collapse.lua

```diff
@@ -270,12 +272,16 @@ local function on_init()
 end
 
 local function on_tick()
-    if game.tick % collapse.speed ~= 0 then
+    if game.tick % collapse.speed ~= 0 or collapse.disabled then
```

Comment:
Collapse "enabled/disabled" is already checked on the function, so this can be removd.

Reply:
Can you tell me where it's checked? As mentioned above my experience was that after a week of the server up it started log spamming about not being able to collapse.

Reply:
Removed, I found another way to do this by calling Collapse.start_now(false)